RUFAS/output_manager.py

```diff
@@ -317,6 +318,29 @@ def _exclude_info_maps(self, pool: Dict[str, pool_element_type]) -> Dict[str, po
                 value.pop("info_maps")
         return pool_copy
 
+    def _load_txt_file_names_to_list(self, dir_path: str) -> List[str]:
+        """ Looks in directory for txt file names and adds them to a list.
+
+        Parameters
+        ----------
+        path : str
+            Path of the directory to be searched.
+
+        Returns
+        -------
+        List
+            A list of txt file names.
+
+        """
```

Comment:
I think a one-liner is enough for this case.
```suggestion
        """ Returns the list of files in the given path"""
```
It's just a suggestion though. If you don't like the one-liner, please add exception to your docstring.

Reply:
I'm fine with a one-liner here if you think it's acceptable.